<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="test.exception_TC">
        <methods>
          <include name="verify_user_able_to_replan_the_trip_in_tripexception_page"/>
          <include name="afterMethod"/>
          <include name="tearDown"/>
          <include name="beforeClass"/>
          <include name="VERIFY_THAT_U3_URL_LAUNCHED_SUCCESSFULLY"/>
        </methods>
      </class> <!-- test.exception_TC -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
